compiler/rustc_middle/src/ty/layout.rs

```diff
@@ -1244,7 +1244,6 @@ pub fn fn_can_unwind(tcx: TyCtxt<'_>, fn_def_id: Option<DefId>, abi: SpecAbi) ->
         | RiscvInterruptS
         | CCmseNonSecureCall
         | Wasm
-        | PlatformIntrinsic
         | Unadjusted => false,
         Rust | RustCall | RustCold | RustIntrinsic => {
             tcx.sess.panic_strategy() == PanicStrategy::Unwind
```

Comment:
This means that `fn_can_unwind` will no longer return `false` on platform intrinsics... but it doesn't seem right to have a completely separate ABI for this? If we truly need them to be considered non-unwinding here we should probably have an attribute for that.

`abi_can_unwind` in ffi_unwind_calls will still return `false` as before. I don't know how it makes sense that these functions disagree on what they say.

Reply:
Hm, but we don't seem to be looking at that attribute here?

Ah never mind. We check the attribute above before coming here.